io.h

```diff
@@ -9,21 +9,27 @@
 #include <SPI.h>
 #include <SdFat.h>
 #include <RHDatagram.h>
+#include <WiFi101.h>
 
+/*
 #define SD_CHIP_SELECT_PIN 10
 #define RH_MESH_MAX_MESSAGE_LEN 60
-#define RFM95_CS 8
-#define RFM95_RST 4
-#define RFM95_INT 3
+#define RFM95_CS 19
```

Comment:
Are these still being used? Hopefully these are now coming out of the config and these defines can be deleted (RFM95_CS, RFM95_RST, RFM_95_INT)

Reply:
Oh, I see all this looks like maybe it is commented out. That's good -- you can go ahead and delete this whole commented section of unused code

Reply:
These are coming from the config. I'll delete the commented code.